project1/index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+
+
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Caesar+Dressing|Metal+Mania|Nunito+Sans|Squada+One&display=swap"
+        rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=PT+Sans+Caption:700&display=swap" rel="stylesheet">
+
+</head>
+
+
+<body>
+    <main>
+        <div class="cut">
+            <h1><span class="Hlight">The Crane Wife</span></h1>
+            <h2>By CJ Hauser</h2>
+            <h3>July 16, 2019</h3>
+        </div>
```

Comment:
To match the suggestion above:
```suggestion
        </header>
```